comment_gh_pr.py

```diff
@@ -0,0 +1,8 @@
+from github import Github
+
+def comment_gh_pr(repo, pr, msg):
+      from os import environ
+      gh = Github(login_or_token=environ['GITHUBTOKEN'], retry=3)
```

Comment:
@gartung , is not it going to break cms bot test? here it assumes that GITHUBTOKEN env is set which in original comment-gh-pr we were using `repo_config.GH_TOKEN`

Reply:
Sorry I forgot I adapted to using an environment variable set from a stored Jenkins secret. I will change it to the CMS convention.